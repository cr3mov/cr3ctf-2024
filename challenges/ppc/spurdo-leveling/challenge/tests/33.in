13898855434624681 995840097921203394
343275 3272490
343275 7901675
