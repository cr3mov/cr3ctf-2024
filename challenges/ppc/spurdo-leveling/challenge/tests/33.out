13341240822615400 86682092279 86682092279
